app/src/main/java/com/example/goodnote/database/LocalDb.kt

```diff
@@ -34,4 +39,21 @@ abstract class LocalDb : RoomDatabase() {
             return Room.databaseBuilder(context, LocalDb::class.java, DB_NAME).build()
         }
     }
+
+    // local db would need scope every time it's initialized, so...
+    private class LocalDbCallback(private val scope: CoroutineScope) : RoomDatabase.Callback() {
+        override fun onCreate(db: SupportSQLiteDatabase) {
+            super.onCreate(db)
+            instance?.let { database ->
+                scope.launch {
```

Comment:
For this operation, you can use `GlobalScope` for this call if you want to perform operation on the IO thread. 
There is no need to provide scope as parameter of the LocalDbCallback constructor.

Something like this:
```   
            GlobalScope.launch {
                  var noteDao = database.noteDao()

                    noteDao.addNote(Note(DUMMY_TITLE, DUMMY_TEXT))
                    noteDao.addNote(Note("epistolary?", DUMMY_TEXT))
                    noteDao.addNote(Note("smth", "lighthouse keepers? village? small island?"))

                }
```